templates/configmap.yaml

```
@@ -19,3 +20,18 @@ data:
   TAIGA_EMAIL_PORT: {{ default 587 .Values.taiga.emailSmtpPort | quote }}
   TAIGA_EMAIL_USER: {{ required "taiga.emailSmtpUser must be set" .Values.taiga.emailSmtpUser | quote }}
 {{- end }}
+{{- if .Values.taiga.ldap.Enabled }}
+  LDAP_AUTH_ENABLE: "True"
+  LDAP_START_TLS: {{ .Values.taiga.ldap.use_tls | default "False" |  quote }}
+  LDAP_SERVER: {{ .Values.taiga.ldap.ldap_server | quote }}
+  LDAP_PORT: {{ .Values.taiga.ldap.port | default 389 | quote }}
+  LDAP_BIND_DN: {{ .Values.taiga.ldap.bind_dn | default "cn=diegoarmandomaradona" |  quote }}
```

I like this default but it'd make more sense to leave it blank if not specified :smile: 
```suggestion
  LDAP_BIND_DN: {{ .Values.taiga.ldap.bind_dn | default "" |  quote }}
```

---

:D completely forgot about this LOL